src/Relude/DeepSeq.hs

```diff
@@ -26,6 +26,10 @@ import Relude.Monad (MonadIO, liftIO, (<$!>))
 
 import qualified Control.Exception.Base (evaluate)
 
+
+-- $setup
+-- >>> import Relude
```

Comment:
I couldn't get `import Relude` to work, would always get:
```
<no location info>: error:
    Could not load module ‘Relude’
    It is a member of the hidden package ‘relude-0.6.0.0’.
    You can run ‘:set -package relude’ to expose it.
    (Note: this unloads all the modules in the current scope.)
```

then follow those instructions and I'd get:
```
<interactive>:1:1: error:
    attempting to use module ‘main:Relude’ (/Users/cmdv/Haskell-OSS/relude/src/Relude.hs) which is not loaded
```
maybe I have things set up wrong 😄

Reply:
Yeah, I can see that it's possible to have invalid config and break `doctest`...

Reply:
yeah not too sure, need to look into it. sorry this was off topic but the PR looks great 🎉

Reply:
Thanks for your great work @Cmdv and @chshersh :trophy: